packages/css/avatar.css

```diff
@@ -0,0 +1,93 @@
+.ds-avatar {
+  --ds-avatar-background: var(--ds-color-accent-base-default);
+  --ds-avatar-color: var(--ds-color-accent-contrast-default);
+  --ds-avatar-size: var(--ds-sizing-12);
+  --ds-avatar-padding: var(--ds-spacing-2);
+  --ds-avatar-border-radius: 50%;
+
+  background-color: var(--ds-avatar-background);
+  height: var(--ds-avatar-size);
+  aspect-ratio: 1 / 1;
+  color: var(--ds-avatar-color);
+  border-radius: var(--ds-avatar-border-radius);
+  overflow: hidden;
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  user-select: none;
+  text-transform: uppercase;
+  text-decoration: none;
+
+  &:has(> svg) {
```

Comment:
Could this be `&:not(:has(img))` so we can simplify the later `&:empty::before`
sizing to become only `width: 100%; height: 100%`? ☺️

Reply:
Doing that breaks in a couple places.
Since we center align the text we only need padding when we have an icon. Doing your suggestion results in this on images:
![image](https://github.com/user-attachments/assets/bb5b22cc-4862-4d4a-84af-43492e62a367)

We can hovever do `&:not(:has(> svg, > img))`

Reply:
Also as a note: We need both img and svg, since consumers can put any svg icon in there

Reply:
Right, so no padding if SVG ☺️